content/images/post-images/2018-9-12/image-x.png

```diff
@@ -0,0 +1 @@
+Dummy file
```

Comment:
Sorry, why is this here?

Reply:
Created to allow me to create a folder in the browser interface. Neglected to delete.  I will remove if preferred.